keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/ManualJudgementNotifier.kt

```diff
@@ -0,0 +1,76 @@
+package com.netflix.spinnaker.keel.echo
+
+import com.netflix.spinnaker.keel.api.ConstraintStatus
+import com.netflix.spinnaker.keel.api.ManualJudgementConstraint
+import com.netflix.spinnaker.keel.api.NotificationConfig
+import com.netflix.spinnaker.keel.api.application
+import com.netflix.spinnaker.keel.echo.model.EchoNotification
+import com.netflix.spinnaker.keel.events.ConstraintStateChanged
+import kotlinx.coroutines.runBlocking
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+
+@Component
+/**
+ * Listens for [ConstraintStateChanged] events where the constraint is a [ManualJudgementConstraint] and sends
+ * out notifications so that users can take action.
+ */
+class ManualJudgementNotifier(
+  private val echoService: EchoService
+) {
+
+  @EventListener(ConstraintStateChanged::class)
+  fun constraintStateChanged(event: ConstraintStateChanged) {
+    // if this is the first time the constraint was evaluated, send a notification
+    // so the user can react via other interfaces outside the UI (e.g. e-mail, Slack)
+    if (event.constraint is ManualJudgementConstraint &&
+      event.previousState == null &&
+      event.currentState.status == ConstraintStatus.PENDING) {
+      event.environment.notifications.map {
+        // TODO: run in parallel
+        runBlocking {
+          echoService.sendNotification(event.toEchoNotification(it))
+        }
+      }
+    }
+  }
+
+  private fun ConstraintStateChanged.toEchoNotification(config: NotificationConfig): EchoNotification {
```

Comment:
We got feedback around the content of this message yesterday - are you planning to update this for this PR, or leave it for a future PR?

Reply:
Future, pretty please! Would like to get a first version of this deployed so more people can play around with it. The message should be easy to change later.